compiler-core/src/javascript/tests/snapshots/gleam_core__javascript__tests__externals__external_type_typescript.snap

```diff
@@ -2,6 +2,6 @@
 source: compiler-core/src/javascript/tests/externals.rs
 expression: "pub type Queue(a)\n\n@external(javascript, \"queue\", \"new\")\npub fn new() -> Queue(a)\n"
 ---
-export type Queue$<I> = any;
+export type Queue$<I> = unknown;
```

Comment:
What are the implications of using `unknown` here?

Reply:
It's a simpler change that improves some of the type-safety issues related to the current `any` type.

It's an improvement because:

- It stops instances of these types, e.g. a `Dict`, from being passed freely anywhere.
- It fixes the problem of strict downstream linters emitting warnings about implicit use of `any`.

But it doesn't address the following:

- Anything can be passed where one of these types is expected. This is because all types are assignable/convertible to `unknown`.
- It's not unique, so e.g. `Dict`, `Regex`, `StringBuilder` are all `unknown` and TS can't tell them apart.

Neither of these are regressions from the current status quo of `any` though, they just aren't improved by it.

The only potential downside I can see with `unknown` is that there _could_ theoretically be TS code using the previous `any` in an incorrect way that will now report a TS error, but that's arguably a positive change to now be able to detect any such mistakes.

Reply:
They're not unknown though, the programmer knows what the external type is in most instances.